index.js

```diff
@@ -24,5 +27,19 @@ module.exports = {
     HSNumber: data.HSNumber,
     /* Helper routines */
     parse: data.parse,
-    dump: data.dump
+    dump: data.dump,
+    graphql: {
+        replace,
+        exprParser,
+        find
+    }
 };
+
+if (typeof window === 'undefined') {
+    // NodeJs
+    module.exports = jsWidesky;
+}
+else {
+    // Browser
+    window.jsWidesky = jsWidesky;
+}
```

Comment:
That makes a lot more sense than duplicating the whole file for a browser-specific version. :-)